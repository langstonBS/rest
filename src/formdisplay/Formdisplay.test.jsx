import React from 'react';
import { shallow } from 'enzyme';
import FormDisplay from './Formdisplay';


describe('FormDisplay component', () => {

  
  it('renders App', () => {
    const wrapper = shallow(< FormDisplay submitType={'put'} url={'safsd'} body={'4'} onChange={jest.fn().mockReturnThis()}/>);
    expect(wrapper).toMatchSnapshot();


  });


});