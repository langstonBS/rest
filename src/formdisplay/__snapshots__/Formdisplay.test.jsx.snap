// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormDisplay component renders App 1`] = `
<Fragment>
  <form>
    <input
      name="url"
      onChange={[MockFunction]}
      type="text"
      value="safsd"
    />
    <div
      className="radio"
    >
      <label>
        <input
          checked={false}
          onChange={[MockFunction]}
          type="radio"
          value="post"
        />
        Post
      </label>
    </div>
    <div
      className="radio"
    >
      <label>
        <input
          checked={false}
          name="get"
          onChange={[MockFunction]}
          type="radio"
          value="get"
        />
        get
      </label>
    </div>
    <div
      className="radio"
    >
      <label>
        <input
          checked={true}
          name="put"
          onChange={[MockFunction]}
          type="radio"
          value="put"
        />
        PUT
      </label>
    </div>
    <div
      className="radio"
    >
      <label>
        <input
          checked={false}
          name="Delete"
          onChange={[MockFunction]}
          type="radio"
          value="Delete"
        />
        Delete
      </label>
    </div>
    <textarea
      name="input"
      type="text"
      value="4"
    />
  </form>
</Fragment>
`;
